<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>800</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1000</width>
    <height>800</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1000</width>
    <height>800</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>微软雅黑</family>
    <pointsize>16</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="TitleLabel">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>220</y>
      <width>461</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>28</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(30, 144, 255);</string>
    </property>
    <property name="text">
     <string>欢迎使用模具管理系统</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="UserLabel">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>315</y>
      <width>60</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>宋体</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>账号:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="PasswdLabel">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>368</y>
      <width>60</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>宋体</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>密码:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="UserEdit">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>312</y>
      <width>120</width>
      <height>35</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{  /*常态*/
	background-color: rgb(204, 221, 255);
	border: none;
	border-radius: 2px;
	color: rgb(0,0,0);
}</string>
    </property>
    <property name="placeholderText">
     <string>请输入账号</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="PasswdEdit">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>365</y>
      <width>120</width>
      <height>35</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{  /*常态*/
	background-color: rgb(204, 221, 255);
	border: none;
	border-radius: 2px;
	color: rgb(0,0,0);
}</string>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
    <property name="placeholderText">
     <string>请输入密码</string>
    </property>
   </widget>
   <widget class="QPushButton" name="RegisterButton">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>475</y>
      <width>80</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{  /*常态*/
	background-color: rgb(30, 144, 255);
	border: none;
	border-radius: 10px;
	color: rgb(255, 255, 255);
}
QPushButton:hover{  /*鼠标悬浮时*/
	background: rgb(49,106,234);
}
QPushButton:pressed{  /*鼠标按下时*/
	background:rgb(96,96,96);
	color: rgb(200, 200, 200);
}
QPushButton:!enabled {   /*按钮禁用时*/
	background: rgb(100, 100, 100);
	color: rgb(200, 200, 200);
}
</string>
    </property>
    <property name="text">
     <string>注册</string>
    </property>
   </widget>
   <widget class="QPushButton" name="LoginButton">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>475</y>
      <width>80</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{  /*常态*/
	background-color: rgb(30, 144, 255);
	border: none;
	border-radius: 10px;
	color: rgb(255, 255, 255);
}
QPushButton:hover{  /*鼠标悬浮时*/
	background: rgb(49,106,234);
}
QPushButton:pressed{  /*鼠标按下时*/
	background:rgb(96,96,96);
	color: rgb(200, 200, 200);
}
QPushButton:!enabled {   /*按钮禁用时*/
	background: rgb(100, 100, 100);
	color: rgb(200, 200, 200);
}
</string>
    </property>
    <property name="text">
     <string>登录</string>
    </property>
   </widget>
   <widget class="QLabel" name="BackGround">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1000</width>
      <height>800</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>1000</width>
      <height>800</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>1000</width>
      <height>800</height>
     </size>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="resources.qrc">:/images/start_back.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="LoginLabel">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>215</y>
      <width>480</width>
      <height>320</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
color: white;
border-radius: 10px;  border: 2px groove lavender;
border-style: outset;
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="LogoLabel">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>710</y>
      <width>241</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="resources.qrc">:/images/logo240x64.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="IPLabel2">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>730</y>
      <width>380</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Microsoft YaHei UI</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(190, 190, 190);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="IPLabel1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>730</y>
      <width>111</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(190, 190, 190);</string>
    </property>
    <property name="text">
     <string>您已连接至：</string>
    </property>
   </widget>
   <widget class="QLabel" name="ShowLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>200</width>
      <height>200</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>370</y>
      <width>85</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>宋体</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>记住密码</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>330</y>
      <width>64</width>
      <height>64</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="resources.qrc">:/images/user.png</pixmap>
    </property>
   </widget>
   <widget class="QPushButton" name="DataBaseButton">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>10</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(204, 221, 255);
border: none;
border-radius: 10px;
color: rgb(255,255,255);</string>
    </property>
    <property name="text">
     <string>数据库管理</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>950</x>
      <y>690</y>
      <width>21</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(190, 190, 190);
border: none;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QComboBox" name="PartCombo">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>415</y>
      <width>200</width>
      <height>35</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QComboBox{  /*常态*/
	background-color: rgb(204, 221, 255);
	border: none;
	border-radius: 2px;
	color: rgb(0,0,0);
}</string>
    </property>
    <property name="editable">
     <bool>false</bool>
    </property>
    <item>
     <property name="text">
      <string>冲压模具寿命管制</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>成型模具寿命管制</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>模具&amp;备件仓储管理</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="PartLabel">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>420</y>
      <width>60</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>宋体</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>功能:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="TimeLabel">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>170</y>
      <width>361</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Microsoft YaHei UI</family>
      <pointsize>26</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>2025-08-05 00:00:00</string>
    </property>
   </widget>
   <widget class="QPushButton" name="LinkButton">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>10</y>
      <width>91</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{  /*常态*/
	background-color: rgb(30, 144, 255);
	border: none;
	border-radius: 10px;
	color: rgb(255, 255, 255);
}
QPushButton:hover{  /*鼠标悬浮时*/
	background: rgb(49,106,234);
}
QPushButton:pressed{  /*鼠标按下时*/
	background:rgb(96,96,96);
	color: rgb(200, 200, 200);
}
QPushButton:!enabled {   /*按钮禁用时*/
	background: rgb(100, 100, 100);
	color: rgb(200, 200, 200);
}
</string>
    </property>
    <property name="text">
     <string>重新连接</string>
    </property>
   </widget>
   <zorder>BackGround</zorder>
   <zorder>LoginLabel</zorder>
   <zorder>UserLabel</zorder>
   <zorder>UserEdit</zorder>
   <zorder>TitleLabel</zorder>
   <zorder>RegisterButton</zorder>
   <zorder>PasswdLabel</zorder>
   <zorder>PasswdEdit</zorder>
   <zorder>LoginButton</zorder>
   <zorder>IPLabel2</zorder>
   <zorder>IPLabel1</zorder>
   <zorder>LogoLabel</zorder>
   <zorder>ShowLabel</zorder>
   <zorder>checkBox</zorder>
   <zorder>label</zorder>
   <zorder>DataBaseButton</zorder>
   <zorder>pushButton</zorder>
   <zorder>PartCombo</zorder>
   <zorder>PartLabel</zorder>
   <zorder>TimeLabel</zorder>
   <zorder>LinkButton</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <tabstops>
  <tabstop>UserEdit</tabstop>
  <tabstop>PasswdEdit</tabstop>
  <tabstop>checkBox</tabstop>
  <tabstop>PartCombo</tabstop>
  <tabstop>RegisterButton</tabstop>
  <tabstop>LoginButton</tabstop>
  <tabstop>DataBaseButton</tabstop>
  <tabstop>pushButton</tabstop>
 </tabstops>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
